ashlar/views.py

```diff
@@ -71,6 +73,16 @@ class BoundaryViewSet(viewsets.ModelViewSet):
     serializer_class = BoundarySerializer
     filter_class = BoundaryFilter
 
+    def create(self, request, *args, **kwargs):
```

Comment:
Prior to this addition, 500s were getting thrown, which made parsing output on the front end a bit more difficult.